src/test/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepositoryTest.java

```diff
@@ -362,6 +374,38 @@ public void whenValidateInvalidCommitCursorsThenException() throws NakadiExcepti
                 });
     }
 
+    @Test
+    public void whenKafkaPublishTimeoutThenCircuitIsOpened() throws Exception {
```

Comment:
@rcillo it is not possible to have the circuit to be opened at the same time every time. For example It depends on the machine you run because Hystrix does not just count number of failures it respect time window in which this failures happened.

In the test I check that at least one failure happened with status=FAILED and details="timed out" which means, for this test case, it was short circuited.

Will it be more appropriate If i replace "timed out" with "short circuited"?